esrally/telemetry.py

```diff
@@ -1707,7 +1707,7 @@ def store_system_metrics(self, node, metrics_store):
 
 
 def store_node_attribute_metadata(metrics_store, nodes_info):
-    # push up all node level attributes to cluster level iff the values are identical for all nodes
+    # push up all node level attributes to cluster level if the values are identical for all nodes
```

Comment:
iff is mathematical notation for "if and only if" that @danielmitterdorfer liked using in Rally but I agree it does not add value here

Reply:
Thank you for this useful context. I decided to revert this comment change for consistency in https://github.com/elastic/rally/pull/1779/commits/81348f48eaafa2d70d69d81d251c84b6ca6b9977.